.github/workflows/tests.yml

```diff
@@ -83,6 +83,9 @@ jobs:
       # composer install cache - https://github.com/ramsey/composer-install
       uses: "ramsey/composer-install@v2"
 
+    - name: Add a template for test to complete
+      run: mkdir -p themes/system/CoreBundle/Resources/views/Override/ && cp .github/ci-files/CoreBundle-Override-index.html.twig  themes/system/CoreBundle/Resources/views/Override/index.html.twig
```

Comment:
I didn't mean to do it as a special github action workflow test. The twig file should live somewhere in the CoreBundle's test directory and the test should move it to the themes/system/... folder and overwrite some existing theme. No need for new route.

Reply:
@escopecz, I need to get this setup via workflow that when I move the file from the `CoreBundle/Tests/Functional/DependencyInjection/Compiler/Template/index.html.twig` to the `themes/system/<path>` via setup method, which executes after the compiler pass completes. So, the override template is not available during the twig cache build. I found no other way than this. I tried composer scaffolding locally, but that did not work on the git-cloned setup.

>and overwrite some existing theme. No need for new route. 

I'll remove the new path then.

Reply:
How about to rebuild the cache?

https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Helper/CacheHelper.php#L25

Reply:
The nuking cache worked locally. I was unaware of this earlier.